core/src/main/java/org/jenkins/ui/icon/IconFormat.java

```diff
@@ -0,0 +1,8 @@
+package org.jenkins.ui.icon;
+
+public enum IconFormat {
+    IMG,
+    EXTERNAL_SVG_SPRITE,
+    INLINE_SVG_SPRITE,
+    INLINE_SVG,
```

Comment:
As far as I can tell these don't do anything currently?

Reply:
They do not, not sure if we should leave them there or remove them for now

Reply:
We should remove if we aren’t implementing them

Reply:
let's do it then